From ed8d086f88eaa59e2d79956488a3af14b924cb09 Mon Sep 17 00:00:00 2001
From: Baozhu Zuo <zuobaozhu@gmail.com>
Date: Sat, 2 Jun 2018 10:29:50 +0800
Subject: [PATCH] remove yuidoc

---
 CMakeLists.txt | 44 ++++++++++++++++++++++----------------------
 1 file changed, 22 insertions(+), 22 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7df2f855..1b633e40 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -125,7 +125,7 @@ if (ANDROID)
   upm_add_compile_flags(CXX -fexceptions)
 endif (ANDROID)
 
-find_package (Threads REQUIRED)
+#find_package (Threads REQUIRED)
 find_package (PkgConfig REQUIRED)
 
 # Force a libmraa search and minimum required version every time a config is generated
@@ -263,9 +263,9 @@ if (CMAKE_VERSION VERSION_LESS "3.1")
   CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
   CHECK_CXX_COMPILER_FLAG("-std=c++0x" COMPILER_SUPPORTS_CXX0X)
   if (COMPILER_SUPPORTS_CXX11)
-    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -pthread")
   elseif (COMPILER_SUPPORTS_CXX0X)
-    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
+    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x -pthread")
   else()
     message(FATAL_ERROR "A C++11 compliant compiler is required to build UPM.")
   endif()
@@ -396,25 +396,25 @@ function (CreateDocumentationTargets)
     endif ()
   endif(BUILDSWIGPYTHON)
 
-  # Check if Yuidoc is installed and add target for API documentation
-  if(BUILDSWIGNODE)
-    find_package (Yuidoc 0.10 REQUIRED)
-    if (YUIDOC_FOUND AND YUIDOC_VERSION VERSION_GREATER "0.10")
-      add_custom_command (
-        OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/jsdoc/jsdoc.stamp
-        COMMAND ${CMAKE_SOURCE_DIR}/doxy/doxygen2jsdoc/docgen.js -m upm -i xml -o jsdoc -t ${CMAKE_CURRENT_SOURCE_DIR}/src -g ../../
-        COMMAND ${YUIDOC_EXECUTABLE} -C --no-sort --helpers ${CMAKE_SOURCE_DIR}/doxy/node/generators/yuidoc/helper.js --themedir ${CMAKE_SOURCE_DIR}/doxy/node/generators/yuidoc/tmpl -o html/node jsdoc/yuidoc/upm
-        COMMAND ${CMAKE_SOURCE_DIR}/doxy/doxygen2jsdoc/tolower.js -i html/node
-        COMMAND cmake -E touch ${CMAKE_CURRENT_BINARY_DIR}/jsdoc/jsdoc.stamp
-        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-        COMMENT "Generating Javascript API documentation with Yuidoc"
-        DEPENDS doc
-        VERBATIM)
-      add_custom_target(jsdoc DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/jsdoc/jsdoc.stamp)
-    else ()
-      message (SEND_ERROR "ERROR - Failed to find a compatible version of Yuidoc. Node.js API doc will not be generated")
-    endif ()
-  endif(BUILDSWIGNODE)
+#  # Check if Yuidoc is installed and add target for API documentation
+#  if(BUILDSWIGNODE)
+#    find_package (Yuidoc 0.10 REQUIRED)
+#    if (YUIDOC_FOUND AND YUIDOC_VERSION VERSION_GREATER "0.10")
+#      add_custom_command (
+#        OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/jsdoc/jsdoc.stamp
+#        COMMAND ${CMAKE_SOURCE_DIR}/doxy/doxygen2jsdoc/docgen.js -m upm -i xml -o jsdoc -t ${CMAKE_CURRENT_SOURCE_DIR}/src -g ../../
+#        COMMAND ${YUIDOC_EXECUTABLE} -C --no-sort --helpers ${CMAKE_SOURCE_DIR}/doxy/node/generators/yuidoc/helper.js --themedir ${CMAKE_SOURCE_DIR}/doxy/node/generators/yuidoc/tmpl -o html/node jsdoc/yuidoc/upm
+#        COMMAND ${CMAKE_SOURCE_DIR}/doxy/doxygen2jsdoc/tolower.js -i html/node
+#        COMMAND cmake -E touch ${CMAKE_CURRENT_BINARY_DIR}/jsdoc/jsdoc.stamp
+#        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+#        COMMENT "Generating Javascript API documentation with Yuidoc"
+#        DEPENDS doc
+#        VERBATIM)
+#      add_custom_target(jsdoc DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/jsdoc/jsdoc.stamp)
+#    else ()
+#      message (SEND_ERROR "ERROR - Failed to find a compatible version of Yuidoc. Node.js API doc will not be generated")
+#    endif ()
+#  endif(BUILDSWIGNODE)
 endfunction()
 
 if (IPK)
-- 
2.11.0

